examples/chart/teleport-cluster/values.yaml

```diff
@@ -3,51 +3,191 @@
 ##################################################
 
 # clusterName is a unique cluster name.
-# This value cannot be changed after your cluster starts.
-# Use fully qualified domain name to access your cluster,
+# This value cannot be changed after your cluster starts without rebuilding it from scratch.
+# We recommend using the fully qualified domain name that you use to access your cluster,
 # for example: teleport.example.com
-clusterName:
+clusterName: ""
 
 ##################################################
 # Values that you may need to change.
 ##################################################
 
-# Version of teleport image, if different from appVersion in Chart.yaml.
+# Version of teleport image, if different from chart version in Chart.yaml.
 teleportVersionOverride: ""
 
 # ACME is a protocol for getting Web X.509 certificates
-# acme enables acme protocol
+# Note: ACME can only be used for single-instance clusters. It is not suitable for use in HA configurations.
+# Setting acme to 'true' enables the ACME protocol and will attempt to get a free TLS certificate from Let's Encrypt.
+# Setting acme to 'false' (the default) will cause Teleport to generate and use self-signed certificates for its web UI.
 acme: false
+# acmeEmail is the email address to provide during certificate registration (this is a Let's Encrypt requirement)
 acmeEmail: ""
+# acmeURI is the ACME server to use for getting certificates. The default is to use Let's Encrypt's production server.
 acmeURI: ""
 
 # Set enterprise to true to use enterprise image
+# You will need to download your Enterprise license from the Teleport dashboard and create a secret to use this:
+# kubectl -n ${TELEPORT_NAMESPACE?} create secret generic license --from-file=/path/to/downloaded/license.pem
 enterprise: false
 
 # If true, create & use Pod Security Policy resources
 # https://kubernetes.io/docs/concepts/policy/pod-security-policy/
 podSecurityPolicy:
   enabled: true
 
-# Set customConfig to true to use a custom config. Create
-# a config resource, name it as release name and place it in the chart namespace.
-customConfig: false
-
 # Labels is a map of key-value pairs about this cluster
 labels: {}
 
+# Mode to deploy the chart in. The default is "standalone". Options:
+# - "standalone": will deploy a Teleport container running auth and proxy services with a PersistentVolumeClaim for storage.
+# - "aws": will deploy a Teleport container running auth and proxy services using DynamoDB for backend/audit log storage and S3 for session recordings. (1)
+# - "gcp": will deploy a Teleport container running auth and proxy services using Firestore for backend/audit log storage and Google storage for session recordings. (2)
+# - "ownconfig": will deploy a Teleport container using a teleport.yaml config file that you provide. (3)
```

Comment:
nit: how about `custom` to match the removed `customConfig` value?

Reply:
https://github.com/gravitational/teleport/pull/6344/commits/03c0485ba5ec880a4a498d58f934df8ef4944a6e